docs/source/models/intervention_models/neonatal/cpap_intervention.rst

```diff
@@ -110,11 +108,31 @@ The following decision tree shows all of the paths from delivery facility choice
 
-When we fill in the location-specific values for delivery facility rates, CPAP coverage, relative risk of mortality with CPAP access, and mortality probability (which is also age-specific), this becomes a system of two linear equations have two unknown (p_RDS_w and p_RDS_wo), which we can solve analytically.
+When we fill in the location-specific values for delivery facility rates, CPAP coverage by facility type, relative risk of mortality with CPAP access, and mortality probability (which is also age-specific), this becomes a system of two linear equations with two unknowns (p_RDS_w and p_RDS_wo), which we can solve analytically.
+
+As mentioned above, it is convenient to model this intervention like a dichotomous risk factor, so that we can reuse the
+:class:`Risk<vivarium_public_health.risks.base_risk.Risk>`
+and :class:`RiskEffect<vivarium_public_health.risks.effect.RiskEffect>` components in Vivarium Public Health,
+rather than having to write new components from scratch.
+Calling CPAP a risk factor can sound a bit confusing because CPAP access is a good thing, so it doesn't sound "risky."
+Instead, we flip it so the risk factor is "*lack* of access to CPAP."
+
+The :code:`RiskEffect` component expects a relative risk (RR) and a population-attributable fraction (PAF).
+Because we are flipping the direction of the risk factor, we need to use the inverse of our original RR, so:
+
+.. math::
+    \text{RR}_{\text{no CPAP}} = \frac{1}{\text{RR}_{\text{CPAP}}}
+
+The PAF is the proportion of deaths due to preterm with RDS that would not occur if all births had access to CPAP.
+Since we use the equation :math:`p(\text{RDS}|\text{CPAP}) = (1 - \text{PAF}_\text{no CPAP}) \cdot p(\text{RDS})`
+in the :code:`RiskEffect` component, we solve for :math:`\text{PAF}_\text{no CPAP}` as follows:
+
+.. math::
+    \text{PAF}_{\text{no CPAP}} = 1 - \frac{p(\text{RDS}|\text{CPAP})}{p(\text{RDS})}
 
-As mentioned above, we can calculate a PAF for "lack of access to CPAP".  This is convenient because Vivarium Public Health includes RiskExposure and RiskEffect components that can represent the way CPAP changes RDS mortality without requiring additional code.
+where the terms on the right hand side can be obtained by solving the system of equations above.
 
-We can derive the PAF and RR of "lack of access to CPAP" from these quantities as follows::
+Here is some pseudocode for deriving the PAF and RR of "lack of access to CPAP" from data::
```

Comment:
Seems like this hardly counts as pseudocode, since all the interesting parts are left as ellipses and comments...

Reply:
Yeah, I feel like all this really adds is the order of operations: first solve the system of equations, then calculate PAF and RR. Maybe this should just be prose and say that. Plus, I'm not sure if others agree, but as I mentioned this morning I think we should solve the system of equations explicitly here.

Reply:
I agree, I think we should solve it explicitly.